File: IOSwiftUI/Backoffice.UI/src/screens/onetimecodes/models/BOOneTimeCodeListResponseModel.ts
import BaseResponseModel from "../../../common/models/BaseResponseModel";
import BOOneTimeCodeModel from "./BOOneTimeCodeModel";

class BOOneTimeCodeListResponseModel extends BaseResponseModel {

    count: number;
    oneTimeCodeList: BOOneTimeCodeModel[];

    constructor() {
        super();

        this.count = 0;
        this.oneTimeCodeList = [];
    }
}

export default BOOneTimeCodeListResponseModel;

File: IOSwiftUI/Common/Messages/OneTimeCodes/BOOneTimeCodeListResponseModel.cs
using System;
using System.Collections.Generic;
using IOSwiftUI.Common.Messages.Base;

namespace IOSwiftUI.Common;

public class BOOneTimeCodeListResponseModel : ResponseModel
{
    public int Count { get; set; }
    public IList<BOOneTimeCodeModel> OneTimeCodeList { get; set; }

    public BOOneTimeCodeListResponseModel(int count, IList<BOOneTimeCodeModel> oneTimeCodeList) : base()
    {
        Count = count;
        OneTimeCodeList = oneTimeCodeList;
    }
}
